src/OrleansRuntime/ConsistentRing/VirtualBucketsRingProvider.cs

```diff
@@ -5,6 +5,8 @@
 
 namespace Orleans.Runtime.ConsistentRing
 {
+    using Orleans.Runtime.Configuration;
```

Comment:
Yeah, I was about to comment in other PRs too. You can configure ReSharper to put outside. I don't really care if inside or outside, but I care about consistency, so having all outside should be the way to go for now